File: MyLanguageDB/src/main/java/ru/herooo/mylanguagedb/entities/Country.java
package ru.herooo.mylanguagedb.entities;

import jakarta.persistence.*;

@Entity
@Table(name="country")
public class Country {
    @Id
    @GeneratedValue
    private long id;

    @Column(name="title")
    private String title;

    @Column(name="code")
    private String code;

    public long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/dto/entity/country.js
export class CountryResponseDTO {
    id;
    title;
    code;

    constructor(countryJson) {
        if (countryJson) {
            this.id = countryJson["id"];
            this.title = countryJson["title"];
            this.code = countryJson["code"];
        }
    }
}